### PR TITLE
Allow searching through virtual field name

### PR DESCRIPTION
**Pull request by yhelen**

### CODE REVIEW

File: app/models/concerns/seasonal.rb [hunk 1]
```diff
@@ -26,6 +26,17 @@ module Seasonal
...
     end
+
+    rails_admin do
+      list do
+        field :year
+        field :student do
+          queryable true
+          searchable [:first_name, :last_name]
+        end
+        include_all_fields
```
Comment (justinkim) [inline app/models/concerns/seasonal.rb]: I don't really understand what this does, can you explain it?

File: app/models/service.rb [hunk 2]
```diff
@@ -8,6 +8,20 @@ class Service < ApplicationRecord
 
   validates :hours, numericality: { greater_than: 0 }, presence: true
 
+  rails_admin do
+    list do
```
Comment (justinkim) [inline app/models/service.rb]: Order here should probably be event, student, hours, I think

File: app/models/concerns/seasonal.rb (hunk 1 above)
Comment (justinkim) [reply app/models/concerns/seasonal.rb]: Oh nvm I got it. It makes it so all default fields are included, even after you explicitly include year and student. 👍

File: app/models/service.rb (hunk 2 above)
Comment (yhelen) [reply app/models/service.rb]: Updated.

**Pull request merged**